.github/sync-repo-settings.yaml

```diff
@@ -162,6 +162,22 @@ branchProtectionRules:
       - 'Kokoro - Test: Java GraalVM Native Image'
       - 'Kokoro - Test: Java 17 GraalVM Native Image'
       - javadoc
+  - pattern: otel-v1-branch
+    isAdminEnforced: true
+    requiredApprovingReviewCount: 1
+    requiresCodeOwnerReviews: true
+    requiresStrictStatusChecks: false
+    requiredStatusCheckContexts:
+    - dependencies (17)
+    - lint
+    - clirr
+    - units (8)
+    - units (11)
+    - 'Kokoro - Test: Integration'
```

Comment:
Does Kokoro need to be updated internally to watch this branch as well?

Reply:
ill double check!